### PR TITLE
task/iam-users provisioning module

### PR DESCRIPTION
**Pull request by sortigoza**
Implements the iam user provisioning module.

Example usage:

```
module "users" {
  source = "./iam-users"

  total_users     = "5"
  tf_state_bucket = "beer-bucket"
}
```

Creates:

Screen Shot 2019-04-23 at 12 31 44 PM
Screen Shot 2019-04-23 at 12 30 57 PM

### CODE REVIEW

File: provision/iam-users/iam-group.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+resource "aws_iam_group" "default" {
```
Comment (charlycastro) [inline provision/iam-users/iam-group.tf:1]: can we add Read only to all other AWS resources?

Comment (sortigoza) [reply provision/iam-users/iam-group.tf:1]: added!

**Review by charlycastro — APPROVED** [review]
nice!

File: provision/main.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
+variable "aws_region" {
+  default = "us-east-2"
+}
+
+provider "aws" {
+  profile = "interviews-provision"
+  region  = "${var.aws_region}"
+}
+
+# terraform {
```
Comment (luisc009) [inline provision/main.tf]: If this commented block is not needed, could you remove it?

**Review by luisc009 — APPROVED** [review]
🚀

File: provision/main.tf (hunk 2 above)
Comment (sortigoza) [reply provision/main.tf]: Yes, actually I'll remove the whole file, it is only for explanatory purposes.

**Pull request merged**